buildSrc/src/main/java/com/osacky/flank/gradle/FladleConfigImpl.kt

```diff
@@ -2,43 +2,44 @@ package com.osacky.flank.gradle
 
 import org.gradle.api.file.RegularFileProperty
 import org.gradle.api.provider.ListProperty
+import org.gradle.api.provider.MapProperty
 import org.gradle.api.provider.Property
 
 data class FladleConfigImpl(
   internal val name: String,
-  override var projectId: String?,
+  override var projectId: Property<String>,
```

Comment:
I am not very familiar with gradle api but if properties are mutable under the  hood you should consider to change `var` -> `val`.